mk/spksrc.wheel.mk

```diff
@@ -14,6 +14,14 @@
 
 WHEEL_COOKIE = $(WORK_DIR)/.$(COOKIE_PREFIX)wheel_done
 
+ifneq ($(filter 1 ON TRUE,$(WHEELS_RUST_BUILD_ENABLE)),)
+## rust specific configurations and install_rust_target
+include ../../mk/spksrc.cross-rust-env.mk
+WHEEL_INSTALL_RUST_TARGET = install_rust_target
+else
+WHEEL_INSTALL_RUST_TARGET =
+endif
```

Comment:
Why not always calling rust instead at toolchain time?  Let's just assume rustc compiler will always be available and part of the toolchain definition as well.

Reply:
I rather would not
- the base is the toolchain
- some packages need specific rust version (nightly instead of stable)
- from other points of view we (always) need cmake, meson, python, dotnet, go, nodejs, ...